### PR TITLE
ENT-8554: Added in-line reasoning for use of Apache modules

### PR DESCRIPTION
**Pull request by nickanderson**
This change is intended to better document and understand our use of Apache
modules in the default configuration.

Merge together:
- https://github.com/cfengine/buildscripts/pull/1039

### CODE REVIEW

**Comment by nickanderson** [comment]
Note, this is just comments.

File: cfe_internal/enterprise/templates/httpd.conf.mustache [hunk 1]
```diff
@@ -24,15 +24,34 @@ LoadModule authz_owner_module modules/mod_authz_owner.so
...
+
+# Our default log format uses features provided by these modules
 LoadModule log_config_module modules/mod_log_config.so
 LoadModule logio_module modules/mod_logio.so
-LoadModule expires_module modules/mod_expires.so
+
+# We use the Header directive which is provided by this module
 LoadModule headers_module modules/mod_headers.so
+
+# We use the BrowswerMatch directive which is provided by this module
```
Comment (craigcomstock) [inline cfe_internal/enterprise/templates/httpd.conf.mustache:36]: s/Browswer/Browser/ ?

**Review by craigcomstock — COMMENTED** [review]
one spelling fix TODO

File: cfe_internal/enterprise/templates/httpd.conf.mustache (hunk 1 above)
Comment (nickanderson) [reply cfe_internal/enterprise/templates/httpd.conf.mustache:36]: ACK, good eye
fixed

**Pull request merged**